### DISCUSSION TITLE
Redirect help

### DISCUSSION THREAD

**Discussion by ryanballa** [vercel/next.js#25570] [Help]
How would I write this redirect rule to only account for numeric values after a c? So matching /foo/c12334 but not /img/cart.img

`{
                source: '/:slug/c:categoryId(.*)',
                destination: '/category/:slug/:categoryId*',
                permanent: true,
}`

**Comment by wwwdepot** [comment]
Try 
```
      {
        source: "/:slug/\\c:id",
        destination: "/prefix/:slug/:id",
        permanent: false,
      },
```
